tools/build/scripts/extract-api-summary.js

```diff
@@ -33,8 +33,10 @@ if (undefined !== argv.gatherFullReport)
   shouldGenerateFullReport = true;
 
 // create output file
-const trimmedApiSignature = (argv.apiSignature.split('.'))[0];
-const sigFileName = path.basename(path.normalize(trimmedApiSignature));
+const apiSignatureNormalized = path.normalize(argv.apiSignature);
+const apiSignatureFileName = apiSignatureNormalized.split('\\').slice(-1)[0];
```

Comment:
`split('\\')` will probably not work on systems other than Windows as they use `/`. Maybe use [`path.parse`](https://nodejs.org/api/path.html#pathparsepath) for this?

Reply:
Changed to: 
```ts
const apiSignatureFileName = path.parse(argv.apiSignature).name;
```